LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
no license = you can't use this code or smth like that i don't really understand i just add a license